src/config.js

```diff
@@ -9,7 +9,8 @@ class Config {
       client_secret,
       currency,
       host,
-      storage
+      storage,
+      secure_fetch
```

Comment:
@agrohs @ynnoj is `secure_fetch` the best name for this param? Would `custom_fetch` work?

Reply:
Yes, I’m not partial as to what it’s called (actually had as something like custom_fetch earlier, but as it is used for the auto call was playing with auth_fetch or secure_fetch or credentials_fetch).

Reply:
Updated name and committed as custom_fetch